pages/api/comments.js

```diff
@@ -0,0 +1,25 @@
+import Cors from 'micro-cors'
+import request from 'request'
+
+const cors = Cors({
```

Comment:
look a bit too permissive? can it be set to allow only from certain domains?

probably not a big deal right now ...

Reply:
Agree. The blog is at blog.dvc.org right? Is CORS needed for different subdomains?

Reply:
Yep, we can configure origins, but this CORS rule is only used for the `/api/comments` route so it is not a big problem in this case.

Reply:
can someone start using us as a proxy for their purposes? 

btw, since this API is not authenticated in any way - what is the conceptual difference between this proxy and using it directly (with CORS enabled in admin for the blog.dvc.org) from the client? You mentioned that API was not designed to be used externally, but from this implementation I don't quite understand the difference to be honest.

Reply:
This is not a proxy. We will not return original content, we will return our own JSON with `{ count: NUMBER }` or `{ error: STRING }`.

Reply:
Difference with direct usage is that original payload is 10kb of json, but we only return one field from it in our API. We can even cache this one number on our side in the future and return it without the need for downloading the original 10kb json every time.

Reply:
kk, got it. 👍 I didn't mean that this is the exact "proxy". I didn't get from the initial conversation that it's only about optimizing 10kb workload, I though that API is not "designed" to be used from the client. Do we actually need to optimize 10kb? Especially considering that we are creating a dependency on the server and should think about rate limits for example and scaling it eventually.

Reply:
I think this is resolved by lines 7 and 8 below which changed after this review comment was made.